test/arithmetic/floating_point/floating_point_test.dart

```diff
@@ -110,4 +110,57 @@ void main() {
 
     expect(checkRan, isTrue);
   });
+
+  test('FP Comparison', () {
+    final fp1 = FloatingPoint32();
+    final fp2 = FloatingPoint32();
+
+    final val1 = FloatingPoint32Value.populator().ofDouble(1.23);
```

Comment:
should this have testing for subnormals, explicit jbit, negative numbers, special NaN stuff, etc?